src/Sentry/Internal/ProcessInfo.cs

```diff
@@ -149,6 +149,9 @@ private static DateTimeOffset GetStartupTime()
     [DllImport("user32.dll", CharSet = CharSet.Auto, ExactSpelling = true)]
     private static extern IntPtr GetForegroundWindow();
 
-    [DllImport("user32.dll", CharSet = CharSet.Auto, SetLastError = true)]
+    // GetWindowThreadProcessId is only available in the Windows SDK, so trying to call this from UWP apps will fail.
+    // We wrap use of this in a try/catch as a workaround. However, we need `ExactSpelling = true` here  to suppress
+    // warnings about the use of this API when compiling UWP applications.
+    [DllImport("user32.dll", CharSet = CharSet.Auto, SetLastError = true, ExactSpelling = true)]
```

Comment:
I get that the warning stated 
```
Please make sure the P/Invoke either points to a Windows API allowed in UWP applications, or a native DLL that is part of the package. 
If for some reason your P/Invoke does not satisfy those requirements, please use [DllImport(ExactSpelling=true) to indicate that you understand the implications of using non-UWP A 
```
but isn't adding `ExactSpelling=true` more a "Trust me, I know what I'm doing" and not a suppression of a warning?
But does it make a difference? The comment explains it.

Reply:
I published a new Sentry.nupkg file after making this change, referenced it in a UWP solution and rebuilt... yes the warnings go away if you do that. It's a very bizarre way to suppress warnings if you ask me (thus the code comment).